core_impl/src/test/java/io/opencensus/implcore/stats/ViewManagerImplTest.java

```diff
@@ -294,12 +299,111 @@ public void testRecordInterval() {
   }
 
   @Test
+  public void testRecordLong_Cumulative() {
```

Comment:
It looks like `testRecordLong_Cumulative` and `testRecordLong_interval` correspond to existing tests for MeasureDouble.  I think it would be better to make them share parameterized methods, for example:

```java
  @Test
  public void testRecordDouble_cumulative() {
      testRecord_Cumulative(MEASURE_DOUBLE, ...);
  }

  @Test
  public void testRecordLong_cumulative() {
      testRecord_Cumulative(MEASURE_LONG, ...);
  }

  private void testRecord_Cumulative(Measure measure, ...) {
    ...
  }
```

Alternatively, you could simplify the tests for long measures, since most of the logic is shared between doubles and longs.  I think it would be enough to test setting and getting a few values in cumulative and interval views, to check that they are rounded properly.

Reply:
Nice suggestion, I've updated a few tests to use parameterized helper methods.